tests/worker/test_workflow.py

```diff
@@ -7081,3 +7066,62 @@ async def test_workflow_priorities(client: Client, env: WorkflowEnvironment):
             task_queue=worker.task_queue,
         )
         await handle.result()
+
+
+@workflow.defn(name="ChildWorkflowInfo")
+class ChildWorkflowInfo:
+    def __init__(self) -> None:
+        self.blocked = True
+
+    @workflow.signal
+    def unblock(self):
+        self.blocked = False
+
+    @workflow.run
+    async def run(self):
+        await workflow.wait_condition(lambda: not self.blocked)
+        return workflow.info().root
+
+
+@workflow.defn(name="WithChildWorkflowInfo")
+class WithChildWorkflowInfo:
```

Comment:
```suggestion
@workflow.defn
class ExposeRootChildWorkflow:
    def __init__(self) -> None:
        self.blocked = True

    @workflow.signal
    def unblock(self):
        self.blocked = False

    @workflow.run
    async def run(self):
        await workflow.wait_condition(lambda: not self.blocked)
        return workflow.info().root


@workflow.defn
class ExposeRootWorkflow:
```

Don't need to manually set name on these, and would rather name the workflow similar to its test if at all possible

Reply:
removed and renamed